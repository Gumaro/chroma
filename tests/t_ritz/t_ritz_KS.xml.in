<?xml version="1.0"?>
<WilsonRitzEigen>
  <Param>
	<version>1</version>
	<Mass>-1.4</Mass>
	<boundary>1 1 1 -1</boundary>
	<nrow>4 4 4 8</nrow>
	<rng>
	   <Seed><elem>11</elem>
	         <elem> 0</elem>
	         <elem> 0</elem>
		 <elem> 0</elem> 
           </Seed>
        </rng> 
  </Param>
  <RitzParams>
	<Neig>8</Neig>
	<Ndummy>4</Ndummy>
	<RsdR>1.0e-5</RsdR>
	<RsdA>1.0e-6</RsdA>
	<RsdZero>1.0e-8</RsdZero>
	<ProjApsiP>false</ProjApsiP>
	<GammaFactor>0.1</GammaFactor>
	<MaxKS>200</MaxKS>
	<MaxCG>10000</MaxCG>
	<MinKSIter>5</MinKSIter>
	<MaxKSIter>200</MaxKSIter>
	<Nrenorm>10</Nrenorm>
   </RitzParams>
   <Cfg>
 	<cfg_type>SZIN</cfg_type>
	<cfg_file>./test_purgaug.cfg1</cfg_file>
   </Cfg>
   <Eigen>
	<eigen_file_stem>./g5_wils_eigen</eigen_file_stem>
	<eigen_volfmt>SINGLEFILE</eigen_volfmt>
   </Eigen>
</WilsonRitzEigen>
