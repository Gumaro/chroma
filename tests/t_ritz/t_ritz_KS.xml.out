<?xml version="1.0"?>


<RitzEigen>
  <ProgramInfo>
    <Setgeom>
      <latt_size>4 4 4 8</latt_size>
      <logical_size>1 1 1 1</logical_size>
      <subgrid_size>4 4 4 8</subgrid_size>
      <total_volume>512</total_volume>
      <subgrid_volume>512</subgrid_volume>
    </Setgeom>
  </ProgramInfo>
  <Input><WilsonRitzEigen>
  <Param>
	<version>1</version>
 <FermionAction>
  <FermAct>UNPRECONDITIONED_WILSON</FermAct>
  <Mass>-1.4</Mass>
 </FermionAction>
	<boundary>1 1 1 -1</boundary>
	<nrow>4 4 4 8</nrow>
	<rng>
	   <Seed><elem>11</elem>
	         <elem> 0</elem>
	         <elem> 0</elem>
		 <elem> 0</elem> 
           </Seed>
        </rng> 
  </Param>
  <RitzParams>
	<Neig>12</Neig>
	<Ndummy>4</Ndummy>
	<RsdR>1.0e-5</RsdR>
	<RsdA>1.0e-6</RsdA>
	<RsdZero>5.0e-6</RsdZero>
	<ProjApsiP>false</ProjApsiP>
	<GammaFactor>0.1</GammaFactor>
	<MaxKS>200</MaxKS>
	<MaxCG>10000</MaxCG>
	<MinKSIter>5</MinKSIter>
	<MaxKSIter>200</MaxKSIter>
	<Nrenorm>10</Nrenorm>
   </RitzParams>
   <Cfg>
 	<cfg_type>SZIN</cfg_type>
	<cfg_file>../test_purgaug.cfg1</cfg_file>
   </Cfg>
   <Eigen>
	<eigen_file_stem>./g5_wils_eigen</eigen_file_stem>
	<eigen_volfmt>SINGLEFILE</eigen_volfmt>
   </Eigen>
</WilsonRitzEigen>

  </Input>
  <Config_info><szin>
      <cfg_version>7</cfg_version>
      <date>Sat Jun 16 00:35:57 2001</date>
      <banner>(Sat Jun 16 00:21:07  2001) Source=MOTIE Target=linux Release= Quarks=g</banner>
      <FermTypeP>0</FermTypeP>
      <Nd>4</Nd>
      <Nc>3</Nc>
      <BetaMC>20.1</BetaMC>
      <BetaMD>0</BetaMD>
      <KappaMC>0</KappaMC>
      <KappaMD>0</KappaMD>
      <MassMC>0</MassMC>
      <MassMD>0</MassMD>
      <dt>0</dt>
      <MesTrj>1</MesTrj>
      <TotalCG>0</TotalCG>
      <TotalTrj>2</TotalTrj>
      <spec_acc>0</spec_acc>
      <NOver>2</NOver>
      <TotalTry>12288</TotalTry>
      <TotalFail>56</TotalFail>
      <Nf>9.48851e+33</Nf>
      <Npf>2</Npf>
      <RefMomTrj>302059064</RefMomTrj>
      <RefFnoiseTrj>36</RefFnoiseTrj>
      <LamPl>9.45468e+33</LamPl>
      <LamMi>0</LamMi>
      <AlpLog>4.2039e-45</AlpLog>
      <AlpExp>4.2039e-45</AlpExp>
      <nrow>4 4 4 8</nrow>
      <seed>
        <Seed>
          <elem>811</elem>
          <elem>1073</elem>
          <elem>1535</elem>
          <elem>1310</elem>
        </Seed>
      </seed>
    </szin>

  </Config_info>
  <Observables>
    <w_plaq>0.902263</w_plaq>
    <s_plaq>0.862579</s_plaq>
    <t_plaq>0.941947</t_plaq>
    <link>0.734641</link>
  </Observables>
<EigSpecRitzKS>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.90287</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.88307</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.872</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.86419</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.89747</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.88941</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.838</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.87792</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.88973</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.91915</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.90098</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.87679</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.89134</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.88423</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.83414</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.85546</final_grad>
    </ev>
    <n_jacob>6</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>6</n_count>
      <final_grad>0.561144</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>0.56443</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.534488</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.528929</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.544029</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.560384</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.564942</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.522848</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.557957</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>0.58014</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.544379</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.544911</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.570272</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.547039</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.577865</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.581684</final_grad>
    </ev>
    <n_jacob>6</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>11</n_count>
      <final_grad>0.161029</final_grad>
    </ev>
    <ev>
      <n_count>11</n_count>
      <final_grad>0.159152</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.142212</final_grad>
    </ev>
    <ev>
      <n_count>12</n_count>
      <final_grad>0.17073</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.143657</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.144834</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.145015</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.148961</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.154278</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.16173</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.166037</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.175633</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.164545</final_grad>
    </ev>
    <ev>
      <n_count>19</n_count>
      <final_grad>0.179556</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.185649</final_grad>
    </ev>
    <ev>
      <n_count>18</n_count>
      <final_grad>0.182318</final_grad>
    </ev>
    <n_jacob>6</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>41</n_count>
      <final_grad>0.0512474</final_grad>
    </ev>
    <ev>
      <n_count>54</n_count>
      <final_grad>0.0497679</final_grad>
    </ev>
    <ev>
      <n_count>43</n_count>
      <final_grad>0.0511901</final_grad>
    </ev>
    <ev>
      <n_count>54</n_count>
      <final_grad>0.0483732</final_grad>
    </ev>
    <ev>
      <n_count>41</n_count>
      <final_grad>0.0480902</final_grad>
    </ev>
    <ev>
      <n_count>41</n_count>
      <final_grad>0.049072</final_grad>
    </ev>
    <ev>
      <n_count>51</n_count>
      <final_grad>0.0464176</final_grad>
    </ev>
    <ev>
      <n_count>41</n_count>
      <final_grad>0.0489347</final_grad>
    </ev>
    <ev>
      <n_count>51</n_count>
      <final_grad>0.0548847</final_grad>
    </ev>
    <ev>
      <n_count>42</n_count>
      <final_grad>0.0501557</final_grad>
    </ev>
    <ev>
      <n_count>55</n_count>
      <final_grad>0.0498336</final_grad>
    </ev>
    <ev>
      <n_count>62</n_count>
      <final_grad>0.053715</final_grad>
    </ev>
    <ev>
      <n_count>51</n_count>
      <final_grad>0.0507215</final_grad>
    </ev>
    <ev>
      <n_count>51</n_count>
      <final_grad>0.0485172</final_grad>
    </ev>
    <ev>
      <n_count>65</n_count>
      <final_grad>0.0565107</final_grad>
    </ev>
    <ev>
      <n_count>60</n_count>
      <final_grad>0.0550967</final_grad>
    </ev>
    <n_jacob>5</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>19</n_count>
      <final_grad>0.0156739</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.0148523</final_grad>
    </ev>
    <ev>
      <n_count>29</n_count>
      <final_grad>0.0151057</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.0146949</final_grad>
    </ev>
    <ev>
      <n_count>32</n_count>
      <final_grad>0.0140153</final_grad>
    </ev>
    <ev>
      <n_count>32</n_count>
      <final_grad>0.0142235</final_grad>
    </ev>
    <ev>
      <n_count>22</n_count>
      <final_grad>0.01558</final_grad>
    </ev>
    <ev>
      <n_count>34</n_count>
      <final_grad>0.0155808</final_grad>
    </ev>
    <ev>
      <n_count>29</n_count>
      <final_grad>0.0143422</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>0.0148391</final_grad>
    </ev>
    <ev>
      <n_count>40</n_count>
      <final_grad>0.0161692</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>0.0152245</final_grad>
    </ev>
    <ev>
      <n_count>81</n_count>
      <final_grad>0.0144945</final_grad>
    </ev>
    <ev>
      <n_count>71</n_count>
      <final_grad>0.0151652</final_grad>
    </ev>
    <ev>
      <n_count>41</n_count>
      <final_grad>0.0170432</final_grad>
    </ev>
    <ev>
      <n_count>34</n_count>
      <final_grad>0.0166385</final_grad>
    </ev>
    <n_jacob>4</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>22</n_count>
      <final_grad>0.00474107</final_grad>
    </ev>
    <ev>
      <n_count>22</n_count>
      <final_grad>0.00460853</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.00453287</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>0.00377115</final_grad>
    </ev>
    <ev>
      <n_count>39</n_count>
      <final_grad>0.00421843</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>0.004284</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>0.00477859</final_grad>
    </ev>
    <ev>
      <n_count>31</n_count>
      <final_grad>0.00483599</final_grad>
    </ev>
    <ev>
      <n_count>38</n_count>
      <final_grad>0.00435594</final_grad>
    </ev>
    <ev>
      <n_count>42</n_count>
      <final_grad>0.00455239</final_grad>
    </ev>
    <ev>
      <n_count>48</n_count>
      <final_grad>0.00489216</final_grad>
    </ev>
    <ev>
      <n_count>46</n_count>
      <final_grad>0.00421957</final_grad>
    </ev>
    <ev>
      <n_count>69</n_count>
      <final_grad>0.00467983</final_grad>
    </ev>
    <ev>
      <n_count>47</n_count>
      <final_grad>0.0047062</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00895729</final_grad>
    </ev>
    <ev>
      <n_count>68</n_count>
      <final_grad>0.00475737</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.00146477</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.00124472</final_grad>
    </ev>
    <ev>
      <n_count>19</n_count>
      <final_grad>0.0014207</final_grad>
    </ev>
    <ev>
      <n_count>17</n_count>
      <final_grad>0.00107552</final_grad>
    </ev>
    <ev>
      <n_count>18</n_count>
      <final_grad>0.00128771</final_grad>
    </ev>
    <ev>
      <n_count>37</n_count>
      <final_grad>0.00127373</final_grad>
    </ev>
    <ev>
      <n_count>24</n_count>
      <final_grad>0.00135838</final_grad>
    </ev>
    <ev>
      <n_count>24</n_count>
      <final_grad>0.00151454</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.00134292</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.00135938</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.00153544</final_grad>
    </ev>
    <ev>
      <n_count>26</n_count>
      <final_grad>0.00131352</final_grad>
    </ev>
    <ev>
      <n_count>37</n_count>
      <final_grad>0.00143882</final_grad>
    </ev>
    <ev>
      <n_count>56</n_count>
      <final_grad>0.00131494</final_grad>
    </ev>
    <ev>
      <n_count>66</n_count>
      <final_grad>0.00246996</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00649508</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.00043305</final_grad>
    </ev>
    <ev>
      <n_count>14</n_count>
      <final_grad>0.000386701</final_grad>
    </ev>
    <ev>
      <n_count>31</n_count>
      <final_grad>0.000327785</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>0.000339621</final_grad>
    </ev>
    <ev>
      <n_count>24</n_count>
      <final_grad>0.000400673</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.000385836</final_grad>
    </ev>
    <ev>
      <n_count>31</n_count>
      <final_grad>0.000415682</final_grad>
    </ev>
    <ev>
      <n_count>34</n_count>
      <final_grad>0.000464075</final_grad>
    </ev>
    <ev>
      <n_count>38</n_count>
      <final_grad>0.000386796</final_grad>
    </ev>
    <ev>
      <n_count>38</n_count>
      <final_grad>0.000429037</final_grad>
    </ev>
    <ev>
      <n_count>34</n_count>
      <final_grad>0.000451011</final_grad>
    </ev>
    <ev>
      <n_count>40</n_count>
      <final_grad>0.000381261</final_grad>
    </ev>
    <ev>
      <n_count>47</n_count>
      <final_grad>0.000431564</final_grad>
    </ev>
    <ev>
      <n_count>39</n_count>
      <final_grad>0.000403595</final_grad>
    </ev>
    <ev>
      <n_count>94</n_count>
      <final_grad>0.000731141</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00464806</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.000135502</final_grad>
    </ev>
    <ev>
      <n_count>29</n_count>
      <final_grad>0.00011167</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>0.000101145</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>0.000105744</final_grad>
    </ev>
    <ev>
      <n_count>26</n_count>
      <final_grad>0.000109361</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.000108412</final_grad>
    </ev>
    <ev>
      <n_count>26</n_count>
      <final_grad>0.000126148</final_grad>
    </ev>
    <ev>
      <n_count>14</n_count>
      <final_grad>0.000146211</final_grad>
    </ev>
    <ev>
      <n_count>26</n_count>
      <final_grad>0.000116103</final_grad>
    </ev>
    <ev>
      <n_count>27</n_count>
      <final_grad>0.00012944</final_grad>
    </ev>
    <ev>
      <n_count>46</n_count>
      <final_grad>0.000133864</final_grad>
    </ev>
    <ev>
      <n_count>37</n_count>
      <final_grad>0.000118371</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.000131175</final_grad>
    </ev>
    <ev>
      <n_count>55</n_count>
      <final_grad>0.000122033</final_grad>
    </ev>
    <ev>
      <n_count>44</n_count>
      <final_grad>0.000225122</final_grad>
    </ev>
    <ev>
      <n_count>62</n_count>
      <final_grad>0.0014389</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>27</n_count>
      <final_grad>3.22638e-05</final_grad>
    </ev>
    <ev>
      <n_count>17</n_count>
      <final_grad>3.21178e-05</final_grad>
    </ev>
    <ev>
      <n_count>35</n_count>
      <final_grad>2.46469e-05</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>3.11168e-05</final_grad>
    </ev>
    <ev>
      <n_count>22</n_count>
      <final_grad>2.9372e-05</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>3.36702e-05</final_grad>
    </ev>
    <ev>
      <n_count>37</n_count>
      <final_grad>3.18314e-05</final_grad>
    </ev>
    <ev>
      <n_count>29</n_count>
      <final_grad>4.43813e-05</final_grad>
    </ev>
    <ev>
      <n_count>42</n_count>
      <final_grad>3.65723e-05</final_grad>
    </ev>
    <ev>
      <n_count>35</n_count>
      <final_grad>3.99641e-05</final_grad>
    </ev>
    <ev>
      <n_count>32</n_count>
      <final_grad>4.04731e-05</final_grad>
    </ev>
    <ev>
      <n_count>49</n_count>
      <final_grad>3.40599e-05</final_grad>
    </ev>
    <ev>
      <n_count>45</n_count>
      <final_grad>4.00708e-05</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>3.60614e-05</final_grad>
    </ev>
    <ev>
      <n_count>55</n_count>
      <final_grad>6.9196e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000679854</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>16</n_count>
      <final_grad>9.884e-06</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>7.94461e-06</final_grad>
    </ev>
    <ev>
      <n_count>13</n_count>
      <final_grad>6.95946e-06</final_grad>
    </ev>
    <ev>
      <n_count>32</n_count>
      <final_grad>9.81516e-06</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>9.16148e-06</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>1.04582e-05</final_grad>
    </ev>
    <ev>
      <n_count>17</n_count>
      <final_grad>9.95275e-06</final_grad>
    </ev>
    <ev>
      <n_count>37</n_count>
      <final_grad>1.33465e-05</final_grad>
    </ev>
    <ev>
      <n_count>31</n_count>
      <final_grad>1.05552e-05</final_grad>
    </ev>
    <ev>
      <n_count>33</n_count>
      <final_grad>1.24138e-05</final_grad>
    </ev>
    <ev>
      <n_count>40</n_count>
      <final_grad>1.28269e-05</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>1.01485e-05</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>1.26708e-05</final_grad>
    </ev>
    <ev>
      <n_count>59</n_count>
      <final_grad>1.09937e-05</final_grad>
    </ev>
    <ev>
      <n_count>76</n_count>
      <final_grad>2.18562e-05</final_grad>
    </ev>
    <ev>
      <n_count>150</n_count>
      <final_grad>0.000200558</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>22</n_count>
      <final_grad>4.28503e-06</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>4.53778e-06</final_grad>
    </ev>
    <ev>
      <n_count>16</n_count>
      <final_grad>4.45117e-06</final_grad>
    </ev>
    <ev>
      <n_count>15</n_count>
      <final_grad>4.72405e-06</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>4.34061e-06</final_grad>
    </ev>
    <ev>
      <n_count>25</n_count>
      <final_grad>4.54033e-06</final_grad>
    </ev>
    <ev>
      <n_count>23</n_count>
      <final_grad>4.6149e-06</final_grad>
    </ev>
    <ev>
      <n_count>24</n_count>
      <final_grad>4.65828e-06</final_grad>
    </ev>
    <ev>
      <n_count>23</n_count>
      <final_grad>4.87829e-06</final_grad>
    </ev>
    <ev>
      <n_count>46</n_count>
      <final_grad>4.31624e-06</final_grad>
    </ev>
    <ev>
      <n_count>22</n_count>
      <final_grad>4.74267e-06</final_grad>
    </ev>
    <ev>
      <n_count>32</n_count>
      <final_grad>4.8662e-06</final_grad>
    </ev>
    <ev>
      <n_count>54</n_count>
      <final_grad>4.56451e-06</final_grad>
    </ev>
    <ev>
      <n_count>27</n_count>
      <final_grad>4.60552e-06</final_grad>
    </ev>
    <ev>
      <n_count>30</n_count>
      <final_grad>6.87944e-06</final_grad>
    </ev>
    <ev>
      <n_count>179</n_count>
      <final_grad>6.28247e-05</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>2.57899e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>2.96472e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>2.82753e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>2.62541e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.51602e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.41943e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.51137e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.71451e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>4.39744e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>2.35277e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.24982e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.46969e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.05347e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>3.33293e-06</final_grad>
    </ev>
    <ev>
      <n_count>7</n_count>
      <final_grad>4.9576e-06</final_grad>
    </ev>
    <ev>
      <n_count>143</n_count>
      <final_grad>1.88594e-05</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <final_n_jacob>1</final_n_jacob>
  <n_cg_tot>6604</n_cg_tot>
  <n_KS>13</n_KS>
  <lambda_H>0.447117 0.44946 0.453732 0.458871 0.463333 0.463896 0.466492 0.467478 0.473042 0.473338 0.474688 0.477214 1 1 1 1</lambda_H>
</EigSpecRitzKS>

  <lambda_Msq>0.447117 0.44946 0.453732 0.458871 0.463333 0.463896 0.466492 0.467478 0.473042 0.473338 0.474688 0.477214 1 1 1 1</lambda_Msq>
  <check_norm>2.57899e-06 2.96472e-06 2.82753e-06 2.62541e-06 3.51602e-06 3.41943e-06 3.51137e-06 3.71451e-06 4.39744e-06 2.35277e-06 3.24982e-06 3.46969e-06</check_norm>
  <check_norm_rel>5.76803e-06 6.59617e-06 6.23171e-06 5.72145e-06 7.58853e-06 7.37111e-06 7.52717e-06 7.94585e-06 9.2961e-06 4.97059e-06 6.84621e-06 7.27072e-06</check_norm_rel>
  <eigFix>
    <lambda_Hw>-0.668668 0.670418 -0.673596 0.677401 0.680686 -0.681099 -0.683002 0.683723 -0.68778 0.687996 -0.688977 0.690807 1 1 1 1</lambda_Hw>
    <n_valid>12</n_valid>
    <valid_eig>1 1 1 1 1 1 1 1 1 1 1 1</valid_eig>
    <check_norm>1.2641e-05 8.04813e-06 2.06928e-05 7.35915e-06 1.84256e-05 3.91838e-05 3.14796e-05 3.63e-05 5.65906e-05 1.07407e-05 4.09942e-05 3.33123e-05</check_norm>
    <check_norm_rel>1.89048e-05 1.20046e-05 3.07199e-05 1.08638e-05 2.70692e-05 5.75303e-05 4.60901e-05 5.30916e-05 8.228e-05 1.56116e-05 5.95001e-05 4.82223e-05</check_norm_rel>
  </eigFix>
<LambdaHighRitz>
  <EigSpecRitzCG>
    <eval1>
      <n_count>86</n_count>
    </eval1>
    <eValues>
      <lambda>-41.7781</lambda>
    </eValues>
    <n_cg_tot>86</n_cg_tot>
  </EigSpecRitzCG>
</LambdaHighRitz>

  <Highest>
    <lambda_Msq_hi>6.4636</lambda_Msq_hi>
  </Highest>
</RitzEigen>